packages/stargate/src/index.ts

```diff
@@ -25,6 +25,17 @@ export {
   AminoMsgWithdrawValidatorCommission,
   AuthExtension,
   BankExtension,
+  createAuthzAminoConverters,
+  createBankAminoConverters,
+  createDistributionAminoConverters,
+  createFeegrantAminoConverters,
+  // Alias for old typo "Freegrant" (https://github.com/cosmos/cosmjs/issues/1258)
+  createFeegrantAminoConverters as createFreegrantAminoConverters,
```

Comment:
Why do you need this line? Is it because of backwards compatibility?

Reply:
Yeah that was the idea. But then again, better remove it now.